src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -137,6 +146,15 @@ export function slickGridDataItemColumnValueWithNoData(value: any, columnDef: an
 	if (displayValue instanceof Array) {
 		displayValue = displayValue.toString();
 	}
+	if (displayValue['ariaLabel']) {
```

Comment:
is this copy paste error? What are you trying to achieve here?

Reply:
slickGridDataItemColumnValueWithNoData is creating cell element. I use it to set ariaLabel for the cell.
Below is the result:
`<div tabindex="-1" role="gridcell" aria-colindex="1" aria-label="Database Icon" class="slick-cell l0 r0"><span class="grid-cell-value-container icon codicon slick-icon-cell-content model-view-component-icon-9">&nbsp;</span></div>`